src/boot_O2_g3/fault.c

```diff
@@ -327,22 +401,23 @@ void osSyncPrintfFPCR(u32 value) {
     }
 }
 
-void Fault_PrintThreadContext(OSThread* t) {
+void Fault_PrintThreadContext(OSThread* thread) {
     __OSThreadContext* ctx;
```

Comment:
```suggestion
    __OSThreadContext* threadCtx;
```
Maybe this was fine as is, but with so many ctx, it's pretty comment to use the prefix